src/com/base/game/inventory/item/ItemType.java

```diff
@@ -1897,6 +2021,45 @@ public String getUseDescription(GameCharacter user, GameCharacter target) {
 		}
 	},
 	
+	BOOK_COW_MORPH(
+			null,
+			false,
+			"Milking Cows",
+			"A book that details cow-morph society.",
+			"book_race_cow_morph",
+			Colour.RACE_COW_MORPH,
+			10,
+			Rarity.LEGENDARY,
+			null,
+			Util.newArrayListOfValues(new ListValue<>(new ItemEffect(ItemEffectType.BOOK_READ_COW_MORPH, null, null)))) {
+
+		@Override
+		public boolean isAbleToBeUsed(GameCharacter target) {
+			if(!Main.game.getPlayer().getRacesAdvancedKnowledge().contains(Race.COW_MORPH)) {
```

Comment:
This probably falls outside of the scope of this PR since all the other books do it like this, but, please, whenever you make an if statement that consists of if(x){ return true } else { return false }, replace it with return x. In this case, that'd be "return !Main.game.getPlayer().getRacesAdvancedKnowledge().contains(Race.COW_MORPH);". I personally feel that explicitly returning true and false as is done now does not add to the readability.